devito/finite_differences/tools.py

```diff
@@ -225,16 +236,19 @@ def make_stencil_dimension(expr, _min, _max):
     return StencilDimension(name='i%d' % n, _min=_min, _max=_max)
 
 
-def symbolic_weights(function, deriv_order, indices, dim):
-    return [function._coeff_symbol(indices[j], deriv_order, function, dim)
+def symbolic_weights(function, deriv_order, indices, x0):
+    return [function._coeff_symbol(indices[j], deriv_order, function, x0)
             for j in range(0, len(indices))]
 
 
 @cacheit
-def numeric_weights(deriv_order, indices, x0):
+def numeric_weights(function, deriv_order, indices, x0):
```

Comment:
Function never gets used here right?

Reply:
It's a standardized signature so can be used through the registry with same inputs for all cases